config/initializers/govuk_formbuilder.rb

```diff
@@ -8,7 +8,7 @@
   # config.default_legend_size: 'm'
   # config.default_legend_tag: nil
   # config.default_caption_size: 'm'
-  # config.default_submit_button_text: 'Continue'
+  config.default_submit_button_text = "Save and continue"
```

Comment:
Should this be an `I18n.t`?

Reply:
I was going to say no - then the word 'Welsh Translation' appeared...